src/envoy/http/jwt_auth/token_extractor.cc

```diff
@@ -55,21 +56,21 @@ JwtTokenExtractor::JwtTokenExtractor(const JwtAuthentication& config) {
     if (use_default) {
       authorization_issuers_.insert(jwt.issuer());
 
-      auto& param_issuers = param_maps_[kParamAccessToken];
+      auto &param_issuers = param_maps_[kParamAccessToken];
       param_issuers.insert(jwt.issuer());
     }
   }
 }
 
 void JwtTokenExtractor::Extract(
-    const HeaderMap& headers,
-    std::vector<std::unique_ptr<JwtTokenExtractor::Token>>* tokens) const {
+    const HeaderMap &headers,
+    std::vector<std::unique_ptr<JwtTokenExtractor::Token>> *tokens) const {
   if (!authorization_issuers_.empty()) {
-    const HeaderEntry* entry = headers.Authorization();
+    const HeaderEntry *entry = headers.Authorization();
     if (entry) {
       // Extract token from header.
-      const HeaderString& value = entry->value();
-      if (StringUtil::startsWith(value.c_str(), kBearerPrefix, true)) {
+      const HeaderString &value = entry->value();
+      if (absl::StartsWith(value.c_str(), kBearerPrefix, true)) {
```

Comment:
remove the third parameter
use `value.getStringView()` for the first parameter.